packages/jellyfish-api-core/src/category/oracle.ts

```diff
@@ -0,0 +1,44 @@
+import { ApiClient } from '../.'
+
+/**
+ * Oracle RPCs for DeFi Blockchain
+ */
+export class Oracle {
+  private readonly client: ApiClient
+
+  constructor (client: ApiClient) {
+    this.client = client
+  }
+
+  /**
+   * Creates an oracle appointment transaction and saves it to the database.
```

Comment:
I know we are copying AIN documentation here, but I think we can do better, we can write something more descriptive like "Creates a price oracle for rely of real time price data".